tests/capi/unittest_capi_service_agent_client.cc

```diff
@@ -1250,6 +1250,52 @@ TEST_F (MLServiceAgentTest, model_gdbus_call_n)
   g_object_unref (proxy_for_model);
 }
 
+/**
+ * @brief Negative testcase of resource gdbus call.
+ */
+TEST_F (MLServiceAgentTest, resource_gdbus_call_n)
+{
+  int ret;
+  gchar *res_info = NULL;
+  GError *error = NULL;
+
+  MachinelearningServiceResource *proxy_for_resource
+      = machinelearning_service_resource_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+          G_DBUS_PROXY_FLAGS_NONE, "org.tizen.machinelearning.service",
+          "/Org/Tizen/MachineLearning/Service/Resource", NULL, &error);
```

Comment:
Can this TC include "dbus-interface.h"?
If so, can you change "org.tizen.machinelearning.service" and "/Org/Tizen/MachineLearning/Service/Resource" to DBUS_ML_BUS_NAME and DBUS_RESOURCE_PATH ?

Reply:
Agreed, I will add new PR for updating pre-defined macro in this testcase later.

Reply:
Added new PR #365 for this.